README.md

```diff
@@ -23,42 +23,38 @@ course use Slack at [tieto-comp-dev](https://tieto-comp-dev.slack.com/messages/)
        subject to change,
     1. You have two weeks since then to create a pull request with all
        the solutions,
-    1. ...but it doesn't there, we expect most of the learning to
-       happen during review process,
+    1. ...but it doesn't end there, we expect most of the learning to
+       happen during review process.
 1. We highly recommend solving the exercise just after the lesson,
 1. Don't hesitate to ask questions and don't take the reviews
    personally.
 
 We hope you will have a lot of fun learning!
 
 ### The Scope
-0. [Lesson 0 - The Setup](./course/lesson_00_the_setup/) -- deadline 2018-02-27
-1. Lesson 1 - Python basics
-1. Lesson 2 - Flow control
-1. Lesson 3 - Functions
-1. Lesson 4 - Unit testing
-1. Lesson 5 - Lists
-1. Lesson 6 - Dictionaries, tuples and sets
-1. Lesson 7 - \*args and \*\*kwargs
-1. Lesson 8 - Manipulating strings
-1. Lesson 9 - Regular expressions
-1. Lesson 10 - Reading and writing files
-1. Lesson 11 - Organizing Files
-1. Lesson 12 - argparse
-1. Lesson 13 - Debugging
-1. Lesson 14 - Keeping Time, Scheduling Tasks, and Launching Programs
```

Comment:
Chapter 15 za bardzo polega na subprocess, threading, requests. IMO zbyt skomplikowane.
Zamiast tego na wyciągłem z listy todo Decorators i dodałem generatory.

Reply:
dobry pomysł, będzie brakować sleepa, ale chyba można bez niego żyć, możemy wywalić ten rozdział i przerzucić datetime gdzie indziej

Reply:
sleep to antipattern IMO. Lepiej nie uczyć w ogóle ;).